src/frequenz/sdk/timeseries/logical_meter/logical_meter.py

```diff
@@ -0,0 +1,111 @@
+"""A logical meter for calculating high level metrics for a microgrid.
+
+Copyright
+Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+License
+MIT
+"""
+
+import asyncio
+import uuid
+from typing import Dict, List
+
+from frequenz.channels import Broadcast, Receiver, Sender
+
+from ...actor import ChannelRegistry, ComponentMetricRequest
+from ...microgrid.component import ComponentMetricId
+from ...timeseries import Sample
+from ._formula_engine import FormulaBuilder, FormulaEngine
+
+
+class LogicalMeter:
+    """A logical meter for calculating high level metrics for a microgrid.
+
+    LogicalMeter instances need to be specified formulas for each of the metrics that
+    need to be calculated.
```

Comment:
This sentence is confusing. A suspect it means that the user needs to specify formulas to a LM.

Reply:
this was also a bit outdated because we've removed the `grid_power`, etc. methods temporarily.  Updated the comment.